README.md

```diff
@@ -66,58 +77,32 @@ If you want to work on improving the components it’s best to run the latest co
     - In the repository of an app do `npm link @nextcloud/vue` (you need to re-link any time you do `npm ci` in the app)
 4. Then build the app with: `npm run build` (or watch for changes with `npm run watch`)
 
-## Translations
+## 🌐 Translations
+
+Use `t` and `n` functions imported from `@nextcloud/l10n` to display translated strings.
 
-This library uses translated strings.
-When you edit/create a translated string, you need to run `npm run l10n:extract` to update the source files.
-Our awesome translation community will then be notified and a bot will sync those changes automatically.
+> Note: these functions are replaced with [`@nextcloud/l10n/gettext`](https://nextcloud-libraries.github.io/nextcloud-l10n/modules/_nextcloud_l10n_gettext.html) on build time.
```

Comment:
This is confusing.
The first part is about functions for nc-l10n which is used in apps, but the note is about this library which uses node gettext, that are two completely different things.

So probably replace the first sentence or restore what was there (only the first sentence) before as that is true.

Reply:
No it is not. Nextcloud l10n is a custom logic that works with server provided translations.

So that is about apps. But this section is about translations IN this library which use a completely different translation strategy.